doc/specs/#163 - Dependencies.md

```diff
@@ -58,7 +58,7 @@ These include other packages. The restriction on these dependencies is that they
 ### External Dependencies
 These include dependencies from outside of the source the original package is distributed. In some cases suitable items may exist in the same source, but for licensing or personal preference, no explicit package should be required. One example is Java. Many vendors offer JREs and JDKs so it may be more reasonable to have a user informed, and allow the user to confirm the presence of a dependency.
 
-
+As a first step the Windows Package Manager should show dependency information to the user, see issue [1012](github.com/microsoft/winget-cli/issues/1012) with [spec](github.com/fzanollo/winget-cli/blob/dependencies/doc/specs/%231012%20-%20Show%20dependencies.md).
```

Comment:
As @jsoref pointed out on the other iteration of this, you should use a relative path, not an absolute.  I believe something like `/doc/specs/%231012%20-%20Show%20dependencies.md` will work, but I am certainly no master of GitHub MD.

Reply:
I'd suggest relative 
```suggestion
As a first step the Windows Package Manager should show dependency information to the user, see issue [1012](github.com/microsoft/winget-cli/issues/1012) with [spec](./%231012%20-%20Show%20dependencies.md).
```

Reply:
Oh nice, thank you @jsoref didn't know you could use links like that on github.